pkg/envtest/komega/default.go

```diff
@@ -0,0 +1,75 @@
+package komega
+
+import (
+	"github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Default is the Komega used by the package global functions.
+// NOTE: Replace it with a custom instance or use WithDefaultClient.
+//       Otherwise the package global functions will panic.
+var Default Komega = &komega{}
```

Comment:
Is the intention here that you can import this and use it in the same way that you would with the default Gomega? Rather than having to have the `Komega` in scope, you just `komega.CreateObject().Should(Succeed())`?

Only nit is that these names don't match the interface at the moment, would it be good to be consistent?

Reply:
The intention for the different names was to be a bit more clear that it's about Kubernetes objects when there is no `komega` in front of it. Also there would be collision with Eventually and Consistently.

Naming isn't great yet though.

Reply:
We can, and that even reduces friction with importing both gomega and komega as `.`.